### PR TITLE
Use MUI for connection controls

### PR DESCRIPTION
**Pull request by WardBrian**
Follow on to #178, this addresses the couple of my comments that were deferred

image

### CODE REVIEW

File: gui/src/app/CompilationServerConnectionControl/ConfigureCompilationServerDialog.tsx [hunk 1]
```diff
@@ -20,103 +29,103 @@ const ConfigureCompilationServerDialog: FunctionComponent<
...
+          <p>
+            The public server is provided for convenience, but may not be as
+            reliable as a local server, depending on the current load and
+            availability.
+          </p>
+        </div>
+      )}
+
+      <Divider />
+      <Button onClick={() => closeDialog()}>Close</Button>
```
Comment (jsoules) [inline gui/src/app/CompilationServerConnectionControl/ConfigureCompilationServerDialog.tsx:128]: Wonder if we really need this, given that there's the `X` and it'll close if you click outside the dialog?

**Review by jsoules — APPROVED** [review]
Two other thoughts to consider:

- There's some weird spacing issues with the "retry" link: vertically, because of the hidden underline, and horizontally, it falls a little too close to the connection status text. I'd consider bumping that out, and possibly just using a `Button` instead of a `Link`.
- If possible, it might be nice to find a way to display the URLs for the public/local connection compilation server. Not a deal-breaker.

I'd be happy to merge this even if you didn't address those, however.

File: gui/src/app/CompilationServerConnectionControl/ConfigureCompilationServerDialog.tsx (hunk 1 above)
Comment (WardBrian) [reply gui/src/app/CompilationServerConnectionControl/ConfigureCompilationServerDialog.tsx:128]: Agreed, removed

**Comment by WardBrian** [comment]
Updated screenshot:
image

**Pull request merged**